rb/lib/selenium/webdriver/remote/w3c/bridge.rb

```diff
@@ -0,0 +1,573 @@
+# encoding: utf-8
+#
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require 'json'
+
+module Selenium
+  module WebDriver
+    module Remote
+      module W3C
+
+        #
+        # Low level bridge to the remote server implementing JSON wire
+        # protocol (W3C dialect), through which the rest of the API works.
+        # @api private
+        #
+
+        class Bridge < Remote::Bridge
+
+          def initialize(capabilities, session_id, **opts)
+            @capabilities = capabilities
+            @session_id = session_id
+            super(opts)
+          end
+
+          def dialect
+            :w3c
+          end
+
+          def commands(command)
+            case command
+            when :status, :is_element_displayed
```

Comment:
These two commands are implemented, but aren't part of w3c spec, so they reference the oss implementation. Probably not necessary to do it this way.